docs/sphinx/sections/api/apidocs/libraries/dagster-airflow.rst

```diff
@@ -14,11 +14,11 @@ Run Airflow on Dagster
 
 .. autofunction:: make_dagster_definitions_from_airflow_dag_bag
 
-.. autofunction:: make_dagster_job_from_airflow_dag
+.. autofunction:: make_schedules_and_jobs_from_airflow_dag_bag
```

Comment:
what's the use case for making this function public?

Reply:
to make it easier to have a definition that include the schedules + jobs from an airflow dagbag but to filter out things that might have been converted natively to dagster or with `load_assets_from_airflow_dag`